LICENSE

```diff
@@ -36,6 +36,7 @@ The following files and directories are exceptions:
 * vendor/assets/javascripts/jquery.jeditable.js is MIT licensed
 * vendor/assets/javascripts/jquery.jnotify.js is ASL 2.0 licensed
 * vendor/assets/javascripts/jstz.js is MIT licensed
+* vendor/assets/javascripts/ipaddr.js is MIT licensed
```

Comment:
Can you make a gem asset out of it? I can do it on your behalf if you think it's too much maintenance. Check out an example here: https://github.com/abenari/spice-html5-rails , it's easy to do, just an engine with the library vendored.

We're exploring new avenues for JS libraries in Foreman, but vendoring more js shouldn't be an option imo. cc @tbrisker @ohadlevy